### ISSUE TITLE
Cannot find symbol build error after latest update

### ISSUE THREAD

**Issue by LadislavBohm** [ammarahm-ed/react-native-mmkv-storage#91]
When you install latest version 0.5.5 there is a file RNMMKVJSIModulePackage with this line:

```
super.getJSIModules(reactApplicationContext, jsContext); // <-- ADD THIS
```

I believe this line should be commented by default because otherwise I am not able to build the app since this class does not extend anything.

Can create PR if you want.

**Comment by hoanglam10499** [comment]
1/ Follow the steps: [https://rnmmkv.vercel.app/#/gettingstarted?id=gt050-installation-steps](url)
2/ After
-> go `node_modules/react-native-mmkv-storage/android/src/java/com/ammarahmed/mmkv`
line 16: `super.getJSIModules(reactApplicationContext, jsContext); // <-- ADD THIS` switch to `comments`
-> go `MainApplication.java`
add `import com.facebook.react.bridge.JSIModulePackage;`

Hope it helps, waiting for PR update.

**Comment by LadislavBohm** [comment]
I know I can comment it myself or create patch-package but wanted to have this confirmed from author/contributor whether the PR I suggested is the way they want to go.

**Comment by ammarahm-ed** [comment]
@LadislavBohm hey, thanks for bringing this to my attention. Indeed that line should be commented out. Comment it out and send a PR.

I will merge and release a new version today that fixes this

**Comment by ammarahm-ed** [comment]
fixed on master.

**Comment by ammarahm-ed** [comment]
released v0.5.6
